### PR TITLE
New package: py3status

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

**Comment by Vaelatern** [comment]
The rule within Void is that for every template altered, there should be one commit. Those commits should be named as directed by the end of the [Manual (link)](https://github.com/voidlinux/void-packages/blob/master/Manual.md#contributing-via-git).

Please do not close this PR just because you are going to make changes. This loses history of what you were asked to do, and spams the developers.

File: srcpkgs/py3status/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'py3status'
+pkgname=py3status
+version=3.0
+revision=1
+short_desc="Alternative i3bar"
+homepage=htpp://i3wm.org
```
Comment (Vaelatern) [inline srcpkgs/py3status/template]: This is not a valid URL. Please also put this just above the `distfiles` variable, near the bottom of the template, for the sake of consistency with other packages.

File: srcpkgs/py3status/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+license=GPL-3
+noarch=yes
+build_style=python-module
+python_versions="2.7"
+pycompile_version="2.7"
```
Comment (Vaelatern) [inline srcpkgs/py3status/template]: Neither this line nor the line before it are necessary

File: srcpkgs/py3status/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'py3status'
+pkgname=py3status
+version=3.0
+revision=1
+short_desc="Alternative i3bar"
+homepage=htpp://i3wm.org
+license=GPL-3
```
Comment (Vaelatern) [inline srcpkgs/py3status/template]: `BSD`. Also add the following function to the bottom of the file:
```
post_install() {
    vlicense LICENSE
}

File: srcpkgs/py3status/template (hunk 2 above)
Comment (teldra) [inline srcpkgs/py3status/template]: You mean: python_versions="2.7" and pycompile_version="2.7"?

I am just not sure, if your comment relates to the last lines of shown code.

File: srcpkgs/py3status/template [hunk 4]
```diff
@@ -3,14 +3,16 @@ pkgname=py3status
...
-pycompile_version="2.7"
 pycompile_module="py3status"
 hostmakedepends="python-setuptools python-devel"
 depends="python"
+homepage=https://github.com/ultrabug/py3status
 distfiles="https://github.com/ultrabug/py3status/archive/${version}.tar.gz"
 checksum=f2c0e9c3ecb0bd1c40e952b16255c97bba184de14629b74c0203feb873a3df21
+
+do_install() {
+	vlicense BSD
```
Comment (Vaelatern) [inline srcpkgs/py3status/template]: See the travis builds for why this is wrong. `vlicense path/to/license/file/with/cwd/being/wrksrc` or in this case `vlicense LICENSE`

File: srcpkgs/py3status/template (hunk 2 above)
Comment (Vaelatern) [inline srcpkgs/py3status/template]: Confirmed.

File: srcpkgs/py3status/template [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+revision=1
+short_desc="Alternative i3bar"
+license=BSD
+noarch=yes
+build_style=python-module
+pycompile_module="py3status"
+hostmakedepends="python-setuptools python-devel"
+depends="python"
+homepage=https://github.com/ultrabug/py3status
+distfiles="https://github.com/ultrabug/py3status/archive/${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/py3status/template]: `${homepage}` here.

**Comment by Vaelatern** [comment]
The rule within Void is that for every template altered, there should be one commit. Those commits should be named as directed by the end of the [Manual (link)](https://github.com/voidlinux/void-packages/blob/master/Manual.md#contributing-via-git).

**Comment by Vaelatern** [comment]
Two things. One: That is the name of the issue, not the name of the commit itself. You have 5 commits here. Two: It is still missing the version.

**Comment by teldra** [comment]
ohoh .. :(

**Comment by Vaelatern** [comment]
Please rebase on master

**Comment by Vaelatern** [comment]
1 commit

**Comment by Vaelatern** [comment]
Why does it not install any files except a license?

File: srcpkgs/py3status/template [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+build_style=python-module
+pycompile_module="py3status"
+hostmakedepends="python-setuptools python-devel"
+depends="python"
+maintainer="xo <oh@rotce.de>"
+homepage=https://github.com/ultrabug/py3status
+distfiles="https://github.com/ultrabug/py3status/archive/${version}.tar.gz"
+checksum=f2c0e9c3ecb0bd1c40e952b16255c97bba184de14629b74c0203feb873a3df21
+
+do_install() {
```
Comment (Vaelatern) [inline srcpkgs/py3status/template]: `post_install()` instead and see what happens

**Comment by Vaelatern** [comment]
Ah.

**Pull request closed without merging**